HW_1/task_2.py

```diff
@@ -0,0 +1,15 @@
+#Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+#Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое 
+#количество журавликов, а Катя сделала в два раза больше журавликов, 
+#чем Петя и Сережа вместе?
+#*Пример:*
+#6 -> 1  4  1
+#24 -> 4  16  4
+#60 -> 10  40  10
+
+a = input("Введите количество журавликов ")
+a = int(a)
+katia = int((a/3)*2)
+petia = int((katia/2)/2)
+serg = int(petia)
```

Comment:
Не помешает бы добавить проверку на кратность 6 входных данных. У математических операций, деления и умножения приоритет одинаковый, то есть скобки не нужны. Кроме того, вместо функций `int` можно использовать целочисленное деление.
```suggestion
katia = a // 3 * 2
petia = serg = katia // 2 / 2
```